docker/Dockerfile.cpu

```diff
@@ -55,6 +55,32 @@ ENV DEVITO_LANGUAGE="openmp"
 # MPICC compiler for mpi4py
 ENV MPICC=$I_MPI_ROOT/bin/mpiicc
 
+##############################################################
+# ICX image
+##############################################################
+FROM base as icx
+
+ # Download the key to system keyring
```

Comment:
This whole part is a waste , just do `FROM icc as icx` and just set the DEVITO env vars